frontend/src/components/contactsList.tsx

```diff
@@ -0,0 +1,90 @@
+import { useEffect, useState } from 'react'
+import { CreateAContactForm } from './contactsForm'
+import axios from 'axios'
+
+interface Contact {
+  id: string
+  firstName: string
+  lastName?: string
+  email?: string
+  phoneNumber?: string
+  notes?: string
+  source?: string
+  ownerID: string
+  relationships?: UserContactRelationship[]
+  importantDateEvent?: ImportantDateEvent[]
+}
+
+interface UserContactRelationship {
+  id: string
+  relationshipType: string
+}
+
+interface ImportantDateEvent {
+  id: string
+  date: string
+  eventType: string
+}
+
+export const ContactsList = () => {
+  const [showCreateAContactForm, setShowCreateAContactForm] = useState<boolean>(false)
+  const [contacts, setContacts] = useState<Contact[]>([])
+
+  const toggleContactForm = () => {
+    setShowCreateAContactForm(!showCreateAContactForm)
+  }
+
+  const handleContactCreate = (newContact: Contact) => {
+    console.log(newContact)
+    setContacts((prevState) => [...prevState, newContact])
+  }
+
+  useEffect(() => {
+    const getContacts = async () => {
+      try {
+        const response = await axios.get('http://localhost:8000/contacts')
```

Comment:
we're missing this route still  I think. If you want to do it in a separate PR can you slap a TODO comment above this line at the very least.

Reply:
I'll do it in this PR.

Reply:
I added the route